instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import Comment from './Comment'
+import './CommentSection.css'
+
+class CommentSection extends Component {
+    constructor(props){
+        super(props);
+        this.state={
+            comments: props.comments,
+            newComment: ''
+        }
+        
+    }
+ 
+ 
+addNewComment = (event,index) => {
```

Comment:
If you include a parameter, but never use it, you can just remove it from the function.